sites/feedback/feedback-docs-fr.html

```diff
@@ -0,0 +1,243 @@
+---
```

Comment:
Please apply the same equivalent changes as requested for the alternate English page.

Reply:
Changed. Thanks